<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>480</width>
    <height>232</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>D2E2I</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QFormLayout" name="options1Layout">
      <property name="rightMargin">
       <number>9</number>
      </property>
      <item row="0" column="0">
       <widget class="QLabel" name="templateFileLabel">
        <property name="text">
         <string>Template file:</string>
        </property>
        <property name="buddy">
         <cstring>templateFileSelectEdit</cstring>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <layout class="QHBoxLayout" name="templateFileSelectLayout">
        <item>
         <widget class="QLineEdit" name="templateFileSelectEdit">
          <property name="baseSize">
           <size>
            <width>999</width>
            <height>0</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="templateFileSelectButton">
          <property name="text">
           <string>Browse...</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="dataFileLabel">
        <property name="text">
         <string>Data file:</string>
        </property>
        <property name="buddy">
         <cstring>dataFileSelectEdit</cstring>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <layout class="QHBoxLayout" name="dataFileSelectLayout">
        <item>
         <widget class="QLineEdit" name="dataFileSelectEdit"/>
        </item>
        <item>
         <widget class="QPushButton" name="dataFileSelectButton">
          <property name="text">
           <string>Browse...</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="outputFolderLabel">
        <property name="text">
         <string>Output folder:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <layout class="QHBoxLayout" name="outputFolderSelectLayout">
        <item>
         <widget class="QLineEdit" name="outputFolderSelectEdit"/>
        </item>
        <item>
         <widget class="QPushButton" name="outputFolderSelectButton">
          <property name="text">
           <string>Browse...</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="options2Layout">
      <item>
       <widget class="QCheckBox" name="generateExcelCheck">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="text">
         <string>Generate Excel Files (.xlsx)</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
        <property name="tristate">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="generateImageCheck">
        <property name="text">
         <string>Generate image files (.png)</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
        <property name="autoExclusive">
         <bool>false</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <spacer name="_verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <layout class="QHBoxLayout" name="bottomLayout">
      <item>
       <widget class="QComboBox" name="languageCombo">
        <property name="sizeAdjustPolicy">
         <enum>QComboBox::AdjustToContents</enum>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="_horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLabel" name="generateLabel">
        <property name="text">
         <string>made by Luiz ðŸ˜Ž</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="generateButton">
        <property name="text">
         <string>Generate</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
